### PR TITLE
MWC 4K 2017: Add article and announcement

### PR DESCRIPTION
**Pull request by Loctav**
It's happening!

### CODE REVIEW

File: news/2017-07-10-osumania-4k-world-cup-2017-registrations-now-open.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
++ We have adjusted the mappool structure and size. Every beatmap is now played under FreeMod condition. HardRock is no eligible mod anymore. Group Stage now has **12** beatmaps to pick from, Round of 16 and Quarterfinals have **13** beatmaps, Semifinals have **14** beatmaps and Finals have **15** beatmaps.
+
+## Prizes
+
+We plan to distribute exclusive and unique merchandise to all our podium winners in different tiers. The items are yet to be announced.
+
++ 1st: $150 per team member, exclusive osu! pins and merchandise, profile badge, "osu!mania Champion" user title for one year
++ 2nd: $80 per team member, exclusive osu! pins and merchandise, profile badge
++ 3rd: $40 per team member, exclusive osu! pins and merchandise, profile badge
```
Comment (TPGPL) [inline news/2017-07-10-osumania-4k-world-cup-2017-registrations-now-open.md]: bold the places? e.g. `**1st:**`

File: news/2017-07-10-osumania-4k-world-cup-2017-registrations-now-open.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## How do I register?
```
Comment (TPGPL) [inline news/2017-07-10-osumania-4k-world-cup-2017-registrations-now-open.md]: maybe just Registration?

File: wiki/Tournaments/MWC/2017/4K/en.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+15. If a player disconnects between the beatmaps and the team can not provide an exchange, the match can be delayed 10 minutes at maximum.
+16. All players and referees must to be treated with respect. Instructions of the referees and tournament management is to be followed. Decisions labeled as final are not to be objected.
+17. Disrupting the match by foul play, picking inappropriate warmup beatmaps (see below), insulting and provoking other players or referees, delaying the match or other deliberate inappropriate misbehavior is strictly prohibited.
+18. The multiplayer chatrooms underlie the [osu! community rules](/wiki/Rules). All chat rules apply to the multiplayer chatrooms, too.
+    - Breaking the chat rules results in a silence. Silenced players can not participate at multiplayer matches and must be exchanged for the time being.
+
+19. In Group stage, 'Win by default' will be considered as win by 4:0, +1.0 score difference ratio.
+20. Unexpected incidences are handled by the tournament management. Referees may allow higher tolerance depending on the given circumstances. This is up to their discretion.
+21. Penalties for violating the tournament rules can be:
```
Comment (TPGPL) [inline wiki/Tournaments/MWC/2017/4K/en.md]: broken bulletlist

File: news/2017-07-10-osumania-4k-world-cup-2017-registrations-now-open.md (hunk 2 above)
Comment (Loctav) [inline news/2017-07-10-osumania-4k-world-cup-2017-registrations-now-open.md]: nah

**Pull request merged**